examples/tv-app/android/java/ContentAppCommandDelegate.cpp

```diff
@@ -91,6 +94,59 @@ void ContentAppCommandDelegate::InvokeCommand(CommandHandlerInterface::HandlerCo
     }
 }
 
+Status ContentAppCommandDelegate::InvokeCommand(EndpointId epId, ClusterId clusterId, CommandId commandId, std::string payload,
+                                                bool & commandHandled, Json::Value & value)
+{
+    if (epId >= FIXED_ENDPOINT_COUNT)
+    {
+        JNIEnv * env = JniReferences::GetInstance().GetEnvForCurrentThread();
+        UtfString jsonString(env, payload.c_str());
+
+        ChipLogProgress(Zcl, "ContentAppCommandDelegate::InvokeCommand send command being called with payload %s", payload.c_str());
+
+        jstring resp =
+            (jstring) env->CallObjectMethod(mContentAppEndpointManager, mSendCommandMethod, static_cast<jint>(epId),
+                                            static_cast<jint>(clusterId), static_cast<jint>(commandId), jsonString.jniValue());
+        if (env->ExceptionCheck())
+        {
+            ChipLogError(Zcl, "Java exception in ContentAppCommandDelegate::sendCommand");
+            env->ExceptionDescribe();
+            env->ExceptionClear();
+        }
+        else
+        {
+            const char * respStr = env->GetStringUTFChars(resp, 0);
```

Comment:
Why not use `src/lib/support/JniTypeWrappers.h` / `JniUtfString` so that you do not worry about releasing?

Reply:
Should we have a failure check in case the Get returns NULL?

Reply:
Replaced with JniUtfString.